tt_metal/llrt/CMakeLists.txt

```
@@ -18,5 +18,5 @@ set(LLRT_SRC
     ${CMAKE_CURRENT_SOURCE_DIR}/blackhole/bh_hal_idle_eth.cpp)
 
 add_library(llrt OBJECT ${LLRT_SRC})
-target_link_libraries(llrt PUBLIC common)
+target_link_libraries(llrt PUBLIC common umd_device)
```

Didn't we make an alias target?  This can (should) now be:

```suggestion
target_link_libraries(llrt PUBLIC common umd::device)
```

Or is this demonstrating pre-CMake-overhaul?

---

Correct, its pre-CMake overhaul.

Minimal change you would need to integrate latest UMD after my PR188 is merged.